docs/04_web/04_integrations/03_React/02_react-complex-application-example.md

```diff
@@ -0,0 +1,22 @@
+---
+title: 'React - Complex Application Example'

```

Comment:
@wjhendry Maybe you have an idea for what this ‘section’ could be called - @skawian isn’t happy with the name of this subpage, and to be honest, I’m not too satisfied with it either 😄 , but I haven’t come up with anything better.

On this subpage, I mainly describe what we have in our repository ```howto-ui-integrations-react```, and how to use it. The repository contains an example React application integrated with all our packages, microfrontends, and PBC.

_There's no rush because before we update this document, we need to update the repository with the code itself._

Reply:
This is the correct way of working - put down the best title you have until you can find a better one.
Are you planning to include a simple application as well? If not, we could just call this:
React: example application